gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -517,17 +546,17 @@ void GazeboRosApiPlugin::onModelStatesDisconnect()
 bool GazeboRosApiPlugin::spawnURDFModel(gazebo_msgs::SpawnModel::Request &req,
                                         gazebo_msgs::SpawnModel::Response &res)
 {
-  // get name space for the corresponding model plugins
+  // get namespace for the corresponding model plugins
   robot_namespace_ = req.robot_namespace;
 
-  // incoming robot model string
+  // incoming entity string
   std::string model_xml = req.model_xml;
 
   if (!isURDF(model_xml))
   {
     ROS_ERROR_NAMED("api_plugin", "SpawnModel: Failure - model format is not URDF.");
     res.success = false;
-    res.status_message = "SpawnModel: Failure - model format is not URDF.";
+    res.status_message = "SpawnModel: Failure - entity format is not URDF.";
```

Comment:
In both cases (spawnURDFModel and spawnSDFModel), you can pass a XML definition of a light instead of a model. That's why the comments were changed from "model" to a more generic "entity", and I initially changed the functions names as well initially (to spawnURDFEntity and spawnSDFEntity).